pulsar-broker/src/test/java/org/apache/pulsar/client/api/SimpleProducerConsumerTest.java

```diff
@@ -1124,7 +1127,7 @@ public void testDeactivatingBacklogConsumer(boolean ackReceiptEnabled) throws Ex
         assertTrue(activeSubscriber.contains(sub2));
     }
 
-    @Test(timeOut = 2000)
+    @Test(timeOut = 5000)
```

Comment:
I'd say to remove these ad-hoc timeouts and just rely on the default 300 seconds timeout.